### PR TITLE
Number of bids updates when new bids are posted

### LINKED ISSUE #1697: Live: Number of bids is incorrect
image

true bid count is 
image

### PR DESCRIPTION
**Pull request by ashfurrow**
This fixes #1697, which isn't _quite_ a showstopper, but it was an easy fix.

### CODE REVIEW

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionStateManager.swift [hunk 1]
```diff
@@ -53,6 +53,7 @@ class LiveAuctionStateManager: NSObject {
         }
 
         socketCommunicator.lotUpdateBroadcasts.subscribe { [weak self] broadcast in
+            print("ws lot broadcast: \(broadcast)")
             self?.stateReconciler.processLotEventBroadcast(broadcast)
```
Comment (orta) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionStateManager.swift]: ^

**Comment by devangt** [comment]
👍

File: Artsy/View_Controllers/Live_Auctions/LiveAuctionStateManager.swift (hunk 1 above)
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/LiveAuctionStateManager.swift]: Left it in to match the other, we've added it several times to help us debug but have never committed it.

**Comment by ashfurrow** [comment]
Removed the `print` statement.

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">Detected some Swift building time outliers</td>
    </tr>
  </tbody>
</table>

Time | Class | Function |
| --- | ----- | ----- |253.1ms | [AuctionViewController.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Auction/AuctionViewController.swift#L183) | @objc dynamic func showRefineTappedAnimated(animated: Bool)
196.2ms | [SwiftyJSON.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Pods/SwiftyJSON/Source/SwiftyJSON.swift#L445) | get {}
170.2ms | [WebSocket.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Pods/Starscream/Source/WebSocket.swift#L528) | private func processRawMessage(buffer: UnsafePointer<UInt8>, bufferLen: Int)
78.2ms | [Observable.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Pods/Interstellar/Sources/Observable.swift#L71) | private final func nextTokenHash() -> Int
76.2ms | [AuctionLotMetadataStackScrollView.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Live_Auctions/Views/AuctionLotMetadataStackScrollView.swift#L18) | required init(viewModel: LiveAuctionLotViewModelType, sideMargin: String)
73.2ms | [LiveAuctionsAdminViewController.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Live_Auctions/LiveAuctionsAdminViewController.swift#L22) | @objc override func viewDidLoad()
64.6ms | [LiveAuctionLotViewController.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Live_Auctions/LiveAuctionLotViewController.swift#L49) | @objc override func viewDidLoad()
63.8ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L25) | func ar_presentBlurredOverlayWithTitle(title: String, subtitle: String, buttonState: BlurredStatusOverlayViewCloseButtonState = default)
61.8ms | [UIViewController+BlurredStatusView.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Core/UIViewController+BlurredStatusView.swift#L31) | (closure)
59.5ms | [RefinementOptionsViewController+Private.swift](/artsy/eigen/tree/ashfurrow-showstopper-fixes/Artsy/View_Controllers/Auction/RefinementOptionsViewController+Private.swift#L63) | func stackView() -> ORStackView


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/danger/danger/">danger</a>
</p>

**Comment by orta** [comment]
Yep 👍

**Pull request merged**